Formula/nnn.rb

```diff
@@ -11,6 +11,8 @@ class Nnn < Formula
     sha256 "26901989e2a66d9ec8806ea21b931e66ab8b6f4a1a35544b4da473a8b490e692" => :el_capitan
   end
 
+  depends_on "readline"
```

Comment:
Why the new dependency on readline?

Reply:
@fxcoudert libedit's readline wrapper is no longer compatible with the code.